accelerator/BUILD

```diff
@@ -70,7 +70,10 @@ cc_library(
         ":mqtt_enable": ["MQTT_ENABLE"],
         "//conditions:default": [],
     }) + select({
-        ":db_enable": ["DB_ENABLE"],
+        ":db_enable": [
+            "DB_ENABLE",
+            "CHRONICLE_ENABLE",
```

Comment:
I am not sure if it makes sense to use more generic terms such as permanode rather than referring Chronicle itself.

Reply:
I think that use permanode is better.